eng/pipelines/runtimelab.yml

```
@@ -51,3 +51,33 @@ jobs:
 # Checkout repository
 #
 - template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build with Debug config and Release runtimeConfiguration
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Linux_x64
+    - Windows_NT_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Debug
+      buildArgs: -s clr+libs+installer -c debug -runtimeConfiguration release
```

```suggestion
      buildArgs: -s clr+libs+installer -c debug -runtimeConfiguration checked
```
to get better coverage.

---

I can add a checked leg but I assume libraries tests should always use release coreclr right?

---

The Utf8String implementation is in CoreLib and all tests for Utf8String are under libraries.

If we were to always use release coreclr with libraries tests, we would get no coverage for debug version of Utf8String, e.g. the asserts in Utf8String would be never exercised.

I think we should have two legs: release everything and debug/checked everything.

---

Done.